### PR TITLE
Fix #564: Create PowerAuth-2023.12.md

### LINKED ISSUE #564: Create PowerAuth-2023.12.md

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

File: docs/PowerAuth-2023.12.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+| PowerAuth Data Adapter | `powerauth-data-adapter.war` | 1.6.0 | Customization component for PowerAuth Web Flow. |
+| PowerAuth Tpp Engine | `powerauth-tpp-engine.war` | 1.6.0 | Third party provider registry and consent engine. |
+
+### Utilities
+
+| Component | Application Name | Version | Description |
+|---|---|---|---|
+| PowerAuth Command Line Tool | `powerauth-java-cmd.jar` | 1.6.0 | Command line tool for integration testing. |
+
+### Mobile Libraries
```
Comment (petrdvorak) [inline docs/PowerAuth-2023.12.md:32]: Consider adding mtoken SDK:

- https://github.com/wultra/mtoken-sdk-ios (1.8.2)
- https://github.com/wultra/mtoken-sdk-android (1.8.3)

**Review by petrdvorak — APPROVED** [review]
Looks OK, one minor comment to consider is about adding mtoken SDK.

File: docs/PowerAuth-2023.12.md (hunk 1 above)
Comment (romanstrobl) [reply docs/PowerAuth-2023.12.md:32]: Good idea, I will add these.

**Review by banterCZ — APPROVED** [review]
Looks good. But shouldn't we also mention the following components?
- Mobile Utility Tool
- User Data Store
- SSL Pinning Utility Tool

**Comment by romanstrobl** [comment]
Ok, I will add these components.

**Review by banterCZ — APPROVED** [review]
Thanks for adding other components.

**Pull request merged**